### PR TITLE
Correct use of tracing error tag

### PR DESCRIPTION
**Pull request by samizzy**
# One-line summary
Use correct tracing tags for errors.

### CODE REVIEW

File: core-services/src/main/java/org/zalando/nakadi/service/publishing/EventPublisher.java [hunk 1]
```diff
@@ -259,9 +258,8 @@ private void validate(final List<BatchItem> batch, final EventType eventType, fi
                 } catch (final EventValidationException e) {
                     item.updateStatusAndDetail(EventPublishingStatus.FAILED, e.getMessage());
                     if (eventType.getCategory() != EventCategory.UNDEFINED) {
-                        TracingService.log(ImmutableMap.of(
-                                "event.id", item.getEvent().getJSONObject("metadata").getString("eid"),
-                                "error", e.getMessage()));
+                        TracingService.setTag("event.id", item.getEvent().getJSONObject("metadata").getString("eid"));
```
Comment (adyach) [inline core-services/src/main/java/org/zalando/nakadi/service/publishing/EventPublisher.java:261]: it should be a log not a tag

File: core-common/src/main/java/org/zalando/nakadi/service/TracingService.java [hunk 2]
```diff
@@ -94,7 +95,7 @@ public static Span setErrorFlag(final Span span) {
 
     public static void logError(final String error) {
         if (error != null) {
-            getActiveSpan().log(ImmutableMap.of("error.description", error));
+            getActiveSpan().log(ImmutableMap.of(ERROR_DESCRIPTION, error));
```
Comment (adyach) [inline core-common/src/main/java/org/zalando/nakadi/service/TracingService.java:98]: should we mark it as error as well ?

**Comment by adyach** [comment]
👍

**Comment by samizzy** [comment]
:+1:

**Pull request merged**